internal/pkg/config/testdata/test-config-alt.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+allowPanicOnTaintedValues: true
+sinks:
+  - packageRE: "^example.com/core"
+    methodRE: "^Sink$"
+  - packageRE: "^example.com/core"
+    methodRE: "^Do$"
+    receiverRE: "^Sinker$"
+fieldTags:
```

Comment:
This doesn't need to be here. This behavior is already adequately tested in the main line of tests.

Reply:
Removed.